snips_nlu_metrics/utils/metrics_utils.py

```diff
@@ -151,16 +154,24 @@ def compute_engine_metrics(engine, test_utterances, intent_list,
         if contains_errors(utterance_metrics, include_slot_metrics):
             if not include_slot_metrics:
                 parsing.pop("slots")
-            errors.append({
+            parsings.append({
+                "is_exact": False,
```

Comment:
I don't think `is_exact` should be returned when `persist_exact_parsings` is False.